multinet/uploaders/csv.py

```diff
@@ -95,6 +101,10 @@ def upload(
 
     # Create table and insert the data
     loaded_table = loaded_workspace.create_table(table, edges)
+
+    if metadata:
+        loaded_table.set_metadata(json.loads(metadata))
```

Comment:
Does this need a try/except to catch invalid JSON?

Reply:
Yep